integrations/zoom.js

```diff
@@ -66,7 +76,9 @@ module.exports = async function(robot, kredits) {
     for(const displayName of names) {
       await createContributionFor(displayName, meetingDetails)
         .then(tx => {
-          robot.logger.info(`[hubot-kredits] Contribution created: ${tx.hash}`);
+          if (tx) {
```

Comment:
What's the case of tx being falsey (maybe makes sense to explain that in a comment)? Btw, I think we can replace the `then` with saving the result in a variable from the `await`, like so:

```js
const tx = await createContributionFor(displayName, meetingDetails);
if (tx) log(foo);
```

Reply:
not sure how to best handle this one: https://github.com/67P/hubot-kredits/pull/53/files#diff-74e18db4da1da4461c9ccb134c582a3eR26

if the contributor is not found then there is no tx.

Reply:
Wait, if the contributor is not found, then the promise wouldn't resolve, would it? In that case it should reject.